adapters/epom/epom.go

```diff
@@ -0,0 +1,172 @@
+package epom
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type EpomAdapter struct {
```

Comment:
Please use the name `adapter`. There is no need to repeat the adapter name from the package and there is no need to export the type (it should start with a lower case letter to be package private).

Reply:
Renamed